haskell-src/Concordium/Types/Accounts.hs

```diff
@@ -262,6 +279,10 @@ instance Serialize effectiveTime => Serialize (StakePendingChange' effectiveTime
 
 type StakePendingChange (av :: AccountVersion) = StakePendingChange' (PendingChangeEffective av)
 
+-- |Coerce a 'StakePendingChange' to use 'Timestamp' for the effective time.
+stakePendingChangeTimestamp :: forall av. (AVSupportsDelegation av, IsAccountVersion av) => StakePendingChange av -> StakePendingChange' Timestamp
```

Comment:
Consider another name for this function, currently it is just the return type.

I had to look up the function a few times to remind me of what it actually does, maybe add `coerce` or something.

Reply:
I cannot think of a very good name for this function, so I will replace it with the definition, since it is so simple.